src/client/actions/Action.js

```diff
@@ -81,17 +81,7 @@ class GenericAction {
   }
 
   getMember(data, guild) {
-    const id = data.user.id;
-    return this.getPayload(
-      {
-        user: {
-          id,
-        },
-      },
-      guild.members,
-      id,
-      PartialTypes.GUILD_MEMBER,
-    );
+    return this.getPayload(data, guild.members, data.user.id, PartialTypes.GUILD_MEMBER);
```

Comment:
Are we sure this change will NOT affect other partials that need this function?

Reply:
The only other Action using this function is GuildMemberRemove, which I've adapted already.